pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -224,14 +224,16 @@ func (b *OFBridge) createGroupWithType(id GroupIDType, groupType ofctrl.GroupTyp
 	return g
 }
 
-func (b *OFBridge) DeleteGroup(id GroupIDType) error {
+func (b *OFBridge) DeleteGroup(id GroupIDType, deleteFlows bool) error {
```

Comment:
Is it used anywhere that `deleteFlows` is true? Another question: why name it `deleteFlows`? I didn't see that this is related to any flows.

Reply:
It is related to flows. In `deleteFlows,` `g.Delete()` is called, which calls `AddOFEntriesInBundle.`

deleteFlows = true is not used in other places. Keeping it as a parameter because I think it is reasonable that flow transactions should be kept at this level. Users can disable it explicitly, not implicitly if the flow transaction is not wanted.

Reply:
How about 'deleteGroupOnOVS'? This parameter is used to decide whether to delete the group on OVS.

Reply:
updated